worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```
@@ -1401,13 +1402,44 @@ public int stackCuboidRegion(Region region, BlockVector3 offset, int count,
                                  boolean copyEntities, boolean copyBiomes, Mask mask) throws MaxChangedBlocksException {
         checkNotNull(region);
         checkNotNull(offset);
+
+        BlockVector3 size = region.getMaximumPoint().subtract(region.getMinimumPoint()).add(1, 1, 1);
+        try {
+            return stackRegionBlockUnits(region, offset.multiply(size), count, copyEntities, copyBiomes, mask);
+        } catch (RegionOperationException ignored) {
+            // Should never be able to happen
+            return 0;
```

Can we just throw an `AssertionError` here? No reason to silently fail.